anvilPoll2024MainAnalysis.Rmd

```diff
@@ -464,7 +474,24 @@ ggsave(here("plots/tooldataresourcecomfortscore_singlepanel.png"), w = 2200, h =
 
 ## Awareness: AnVIL Demos
 
-**Takeaway:**
+**Takeaway:** Most respondents have not attended an AnVIL Demo. To investigate whether this is an awareness issue, we aggregated all responses except `No, didn't know of`. We see that the majority of respondents are aware of AnVIL Demos. These responses are just distributed among different ways of utilizing the demos. Further, there's awareness among both current and potential AnVIL users.
+
+### Raw responses
+
+```{r echo=FALSE, message=FALSE}
+demoPlotRaw <- resultsTidy %>%
+  group_by(UserType, AnVILDemo) %>%
+  summarize(count = n()) %>%
+  ggplot(aes(y=reorder(AnVILDemo, count),
+             x = count,
+             fill = UserType)) +
+  geom_bar(stat = "identity") +
+  ggtitle("Have you attended a monthly AnVIL Demo?")
+
+stylize_bar(demoPlotRaw)
+```  
+
+### Responses recoded to focus on awareness
 
 ```{r}
```

Comment:
Might want to set up default chunk options at the top to not show code, eg:
````
```{r, setup, include=FALSE}
knitr::opts_chunk$set(
  message = FALSE, echo = FALSE, warning = FALSE
)
```
````